src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/OtlpGrpcExportClient.cs

```diff
@@ -158,6 +172,27 @@ public override ExportClientResponse SendExportRequest(byte[] buffer, int conten
         }
     }
 
+    protected override Stream Compress(Stream dataStream)
+    {
+        var compressedStream = new MemoryStream();
+
+        compressedStream.WriteByte(1);
+        compressedStream.Write(new byte[4], 0, 4);
```

Comment:
Rather than allocate an empty 4 byte buffer, can we just write `0` 4 times? Alternatively, `[0, 0, 0, 0]` might allow the compiler to be more efficient with regards to the allocation and use a static shared buffer for all calls into the method.